test/test-utils/helpers/overrides-helper.ts

```diff
@@ -54,6 +54,11 @@ export class OverridesHelper extends GameManagerHelper {
    * @defaultValue `true`
    */
   public normalizeNatures: boolean = true;
+  /**
+   * If `true`, will automatically set the level cap to `1` at the start of each test (effectively disabling level ups).
+   * @defaultValue `true`
+   */
+  public disableLevelUp: boolean = true;
```

Comment:
shouldn't it be named disableExpGain instead?
```suggestion
   * If `true`, will automatically set the level cap to `1` at the start of each test
   * (effectively disabling experience gain, and thus, level ups).
   * @defaultValue `true`
   */
  public disableExpGain: boolean = true;
```

Reply:
Yeah that seems more precise.